packages/host/app/components/operator-mode/stack-item.gts

```diff
@@ -412,16 +413,25 @@ export default class OperatorModeStackItem extends Component<Signature> {
     this.containerEl.scrollTop = 0;
   });
 
-  private doCloseAnimation = dropTask(async () => {
-    this.isClosing = true;
-    if (this.itemEl) {
-      await new Promise<void>((resolve) => {
-        scheduleOnce('afterRender', this, this.handleCloseAnimation, resolve);
-      });
-    }
+  private async startAnimation(animationFlag: 'isClosing' | 'isMovingForward') {
+    this[animationFlag] = true;
+    if (!this.itemEl) return;
+
+    return new Promise<void>((resolve) => {
+      scheduleOnce(
+        'afterRender',
+        this,
+        this.handleAnimationCompletion,
+        resolve,
+      );
+    });
+  }
+
+  private doAnimation = task(async (type: 'isClosing' | 'isMovingForward') => {
```

Comment:
Is there any value to this being a task when it is just delegating to one async method?

Reply:
Actually no - I will take that out